app/javascript/packs/application.js

```diff
@@ -47,11 +48,23 @@ class App extends Component {
               path="/appointments/:id/check_in"
               component={CheckIn}
             />
+            <Route
+              path="/users/sign_out"
+            />
           </div>
         </div>
       </Router>
     );
   }
+
+  _signOut() {
+    $.ajax({
```

Comment:
What it looks like is happening is that, since we aren't using jQuery, `$` is undefined so we're getting an error and the link is falling back on its default behavior: navigating to `/users/sign_out` with a `GET` request — fittingly, this is _exactly_ what this function seems to be intended to do. 😀

For our communication with the server, we use [`window.fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) (the linked page includes a pretty great comparison of `fetch` vs `$.ajax`, too). Lemme know if you have any questions about it.